### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by jannyHou**
### Description

Fixing 2 failures in https://cis-jenkins.swg-devops.com/job/ds/job/loopback-connector-mysql~master/130/console
#### Related issues

- connect to <link_to_referenced_issue>

### Checklist

- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: test/datetime.test.js [hunk 1]
```diff
@@ -39,14 +39,15 @@ describe('MySQL datetime handling', function() {
     setConnectionTimezones('Z');
   });
   after(function(done) {
+    console.log('after');
```
Comment (raymondfeng) [inline test/datetime.test.js]: Please remove console.log.

File: test/transaction.promise.test.js [hunk 2]
```diff
@@ -176,7 +176,8 @@ describe('transactions with promise', function() {
     it('should rollback the transaction if timeout', function(done) {
       Post.find({where: {title: 't3'}}, {transaction: currentTx},
         function(err, posts) {
-          if (err) return done(err);
+          should.exist(err);
```
Comment (raymondfeng) [inline test/transaction.promise.test.js]: This change is suspicious. We expect to rollback an active transaction upon timeout.

Comment (jannyHou) [reply test/transaction.promise.test.js]: @raymondfeng yeah I know:
My local tests with mysql 5.6.17 passes this test, **only CI fails**, that's why I make this change.

And according to [this line](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/dao.js#L153) and [the ensureActive method](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/transaction.js#L201), an error should be thrown here. 

Any suggestion?

**Comment by jannyHou** [comment]
@slnode test please

File: test/transaction.promise.test.js (hunk 2 above)
Comment (jannyHou) [reply test/transaction.promise.test.js]: never mind I think it's related to the `--exit` flag :p

File: package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "pretest": "node pretest.js",
     "lint": "eslint .",
-    "test": "mocha --timeout 10000  test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js",
+    "test": "mocha --timeout 10000 --exit test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js",
```
Comment (bajtos) [inline package.json]: @jannyHou please move this flag to https://github.com/strongloop/loopback-connector-mysql/blob/master/test/mocha.opts

**Pull request merged**